index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <link rel="stylesheet" href="./assets/css/style.css" />
+        <title>Elizabeth's Portfolio Page</title>
+    </head>
+  <body>
+      <!--navigation bar of the web page-->
+      <header class="header">
+      <h1>Elizabeth's Portfolio</h1>
+      <nav class=nav-container> 
+      <div>
+        <button type="email"> <img src="images/designes/EmailIcon.jpg" class="email"> Email</button>
+        <a href="Name"> 
+      </div>
+      <div>
+        <button type="GitHub"> <img src="images/designes/GitHubIcon.gif" class="github">Git Hub</button>
```

Comment:
It's tricky to add a link to a `<button>` element. You are not able to add a `href` attribute unfortunately so currently these buttons aren't doing anything. I would advise to change these to `<a>` tags, add your links with the `href` attribute and then style as you wish with CSS (add a class name)